src/arch/aarch64/CodeGen.zig

```diff
@@ -4198,10 +4199,20 @@ fn airArg(self: *Self, inst: Air.Inst.Index) !void {
     return self.finishAir(inst, result, .{ .none, .none, .none });
 }
 
+fn airTrap(self: *Self) !void {
+    // Reference: https://github.com/llvm/llvm-project/blob/7ecb5145bae57fce44c53f3e8eacea76078e3825/llvm/lib/Target/AArch64/AArch64FastISel.cpp#L3603-L3613
+    _ = try self.addInst(.{
+        .tag = .brk,
+        .data = .{ .imm16 = 0x1000 },
+    });
+    return self.finishAirBookkeeping();
+}
+
 fn airBreakpoint(self: *Self) !void {
+    // Reference: https://github.com/llvm/llvm-project/blob/7ecb5145bae57fce44c53f3e8eacea76078e3825/llvm/lib/Target/AArch64/AArch64FastISel.cpp#L3603-L3613
     _ = try self.addInst(.{
         .tag = .brk,
-        .data = .{ .imm16 = 1 },
+        .data = .{ .imm16 = 0xf000 },
```

Comment:
If you look at the references, it seems this was "wrong" previously, or at least didn't match LLVM. They probably have some solid opinions on this stuff so I changed this to do what they do.

Reply:
I believe this is a standard value used for breakpoint on all OSes. Here's Windows for instance https://devblogs.microsoft.com/oldnewthing/20220822-00/?p=107032

Reply:
Yup, looks about right:
```
        brk     #0xf000     ; breakpoint
```
So this is good as-is right?